packages/govuk-frontend/src/govuk/components/exit-this-page/exit-this-page.puppeteer.test.js

```diff
@@ -45,7 +45,7 @@ describe('/components/exit-this-page', () => {
     })
 
     it('navigates to the href of the skiplink', async () => {
-      await goToExample(page, 'exit-this-page-with-skiplink')
+      const page = await goToExample(browser, 'exit-this-page-with-skiplink')
```

Comment:
We still have a few Review app "Example tests" alongside our "Component tests" to relocate:

* https://github.com/alphagov/govuk-frontend/pull/4419

I've updated `goToExample()` and `goToComponent()` to make their own (unmocked) Puppeteer pages to avoid these "Example tests" from potentially hitting a mocked network response

### Before
Passes in the global `page` object which might be mocked

```mjs
await goToComponent(page, componentName)
await goToExample(page, exampleName)
```

### After
Passes in the global `browser` object to create a new page

```mjs
await goToComponent(browser, componentName)
await goToExample(browser, exampleName)
```